langchain4j-vertex/src/test/java/dev/langchain4j/model/vertex/VextexAiLanguageModelIT.java

```diff
@@ -0,0 +1,50 @@
+package dev.langchain4j.model.vertex;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class VextexAiLanguageModelIT {
+
+    @Test
+    @Disabled("To run this test, you must have provide your own endpoint, project and location")
+    void testLanguageModel() {
+        VextexAiLanguageModel vextexAiLanguageModel = new VextexAiLanguageModel(
```

Comment:
no need to test it via ctor, logic should be the same

Reply:
ok, deleted